client/galaxy/scripts/components/Workflow/Editor/EditorPanel.vue

```diff
@@ -6,11 +6,11 @@
             </div>
         </div>
         <div class="unified-panel-body workflow-right">
-            <div v-if="canvas" class="m-1">
+            <div class="m-1" v-show="canvas">
```

Comment:
Since canvas is a prop that won't be changing dynamically, v-if makes more sense here unless I'm missing something.  v-show still renders, it just sets display:hidden.  We don't even want to render if canvas is falsy

Reply:
Canvas is a prop which changes dynamically, but in principle we would still prefer `v-if`. In this case, we want to avoid that the forms are being destroyed when switching to the workflow reports view. At least while they are being populated into the panel by a legacy utility function.

Reply:
Ah-ha!  The forms, okay, yeah -- those are pretty heavyweight and we should preserve the elements.  Makes sense, thanks!

Reply:
Thank you for the review.